### PR TITLE
feat: Node.js Dev Doc(Official)

### PR DESCRIPTION
**Pull request by NarendraKoya999**
## Related Issue

Closes: #251 

## Description of Changes
- **Updated `Node.js Dev Doc (Official) in  frameworks.astro File` for Learning Node.js PRO Level**
## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

|      Original       |       Updated        |
| :-----------------: | :------------------: |
| Original screenshot | Updated screenshot |

Please provide any necessary screenshots to illustrate the changes made.

### CODE REVIEW

**Comment by NarendraKoya999** [comment]
@bhartik021 . Please Review on My PR

**Review by Alkaison — APPROVED** [review]
All good!

File: src/pages/frameworks.astro
Comment (Alkaison) [inline src/pages/frameworks.astro:1]: It has merge conflicts and also I noticed that the resource has been added under a recent PR, try adding some new resource for Node.js.

**Review by Alkaison — CHANGES_REQUESTED** [review]

File: src/pages/frameworks.astro
Comment (NarendraKoya999) [reply src/pages/frameworks.astro:1]: @Alkaison . It's Different Thing. This Resource is helpful for Continue Learning of JavaScript along with Node.js from Very Basic Level to Advanced Level.

**Pull request merged**